lib/fabrik/DockerInstance.js

```diff
@@ -7,6 +7,9 @@ const errors = require('../errors');
 const utils = require('../utils');
 const docker = require('../docker');
 const BaseInstance = require('./BaseInstance');
+const CONST = require('../constants');
+const CfPlatformManager = require('./CfPlatformManager');
```

Comment:
DockerInstance referring both platformManagers? It breaks the purpose of having platform managers.

Reply:
It is to instantiate the classes but this.platformManager having the actual value.